Python/Product/Uap/Interpreter/PythonUapInterpreter.cs

```diff
@@ -0,0 +1,237 @@
+﻿/* ****************************************************************************
+ *
+ * Copyright (c) Microsoft Corporation. 
+ *
+ * This source code is subject to terms and conditions of the Apache License, Version 2.0. A 
+ * copy of the license can be found in the License.html file at the root of this distribution. If 
+ * you cannot locate the Apache License, Version 2.0, please send an email to 
+ * vspython@microsoft.com. By using this source code in any fashion, you are agreeing to be bound 
+ * by the terms of the Apache License, Version 2.0.
+ *
+ * You must not remove this notice, or any other, from this software.
+ *
+ * ***************************************************************************/
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PythonTools.Analysis;
+using Microsoft.PythonTools.Interpreter;
+using Microsoft.VisualStudioTools;
+
+namespace Microsoft.PythonTools.Uap.Interpreter {
+    class PythonUapInterpreter : IPythonInterpreter, IPythonInterpreterWithProjectReferences2, IDisposable {
```

Comment:
The default will interrogate the DLL (PYD) on the debugger machine.  Right now, the DLL cannot be loaded outside an app, hence this prevents the dll from being loaded and interrogated.  I believe this is what the default interpreter does.